demos/security_barrier_camera_demo/main.cpp

```diff
@@ -160,12 +161,16 @@ struct Context {  // stores all global data for tasks
         std::chrono::steady_clock::time_point prevShow;  // time stamp of previous imshow
         std::map<int64_t, GridMat> gridMats;
         std::mutex drawerMutex;
+        std::ostringstream outThroughput;
+        unsigned framesAfterUpdate;
+        std::chrono::steady_clock::time_point localT0;
```

Comment:
It's not really local at this point. I think `updateTime` would be a good name, since it would mesh well with `framesAfterUpdate`.

Reply:
Renamed